pkg/deb/debian/control.base

```diff
@@ -6,7 +6,7 @@ Build-Depends: g++, cmake, dpkg-dev, libyajl-dev (>= 2.0), libssl-dev, libboost1
 
 Package: stackdriver-metadata
 Architecture: amd64
-Depends: libyajl2, libboost-filesystem1.55.0, libboost-program-options1.55.0, libboost-system1.55.0, libboost-thread1.55.0
+Depends: libyajl2, libboost-filesystem1.55.0, libboost-program-options1.55.0, libboost-system1.55.0, libboost-thread1.55.0, ca-certificates
```

Comment:
Update the readme to include this new dependency

Reply:
Updated.